README.md

```diff
@@ -49,9 +49,11 @@ Using Docker on Windows will also need a couple of additional configurations bec
 
 By default, Neo4j requires authentication and requires us to first login with neo4j/neo4j and set a new password. We will skip this password reset by initializing the authentication none when we create the Docker container using the --env NEO4J_AUTH=none.
 
-3. After running the above command, neo4j instance will be created and container starts running, we can verify the same by accessing neo4j browser(http://localhost:7474/browser).
+3. Load seed data to neo4j using the instructions provided in the [link](https://github.com/project-sunbird/knowledge-platform/tree/master-data/master-data/loading-seed-data.md)
```

Comment:
Use relative path of the file every where.

Reply:
Made suggested changes.